profiler/src/ProfilerEngine/Datadog.Profiler.Native/SystemTime.cpp

```diff
@@ -0,0 +1,46 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
```

Comment:
This seems specific to windows.
We should have a `std::uint64_t GetCpuTime(<type> threadId)` as interface.
Linux and Windows should implement their own in their own project

Reply:
I'm not sure about specific interface that would require an implementation kept as a service or any lifetime management... instead, maybe define a pal.h file with multi-platform code.
I would keep as it is until we decide to gather all other Linux/Windows code in a common place